env/env.go

```diff
@@ -59,6 +76,25 @@ func Init(c *cli.Context) {
 	}
 }
 
+func ListEnvs(c *cli.Context) error {
```

Comment:
ideally use this API https://github.com/temporalio/cli/blob/20765bccf46598ecd6572637d7091c5bae07d48b/taskqueue/task_queue_commands.go#L91

It reads `--output` option that allows to set JSON or Table view (as well as few other options)

Reply:
then you want to add this option to list env command https://github.com/temporalio/cli/blob/20765bccf46598ecd6572637d7091c5bae07d48b/taskqueue/task_queue.go#L46

Reply:
you may want to use PrintItems API for both verbose and non-verbose cases

Reply:
hi @feedmeapples , thank you for your input.

are your referring to `Value:    string(output.Table),` option here: https://github.com/temporalio/cli/blob/20765bccf46598ecd6572637d7091c5bae07d48b/taskqueue/task_queue.go#L50

Reply:
referring to the whole `--output` Flag
```
&cli.StringFlag{
					Name:     output.FlagOutput,
					Aliases:  common.FlagOutputAlias,
					Usage:    output.UsageText,
					Value:    string(output.Table),
					Category: common.CategoryDisplay,
				},
```

If we add it to `env list` command it will support JSON with 
`temporal env list --output json --v`  (should work without the -v too as long as PrintItems API is used)

Reply:
@feedmeapples I see now, thank you. Okay then code needs to change. My only concern now is to dynamically build the struct for the key value pairs to show properly.

Reply:
@feedmeapples  The best I could come up with is the following changeset: https://github.com/temporalio/cli/pull/211/commits/304809b856cded4d8647eb1319c5f29dde9dee49 

Which brings the output to:

```

❯ ./temporal env list
  cloud
  default
  dev
  local


❯ ./temporal env list -v
   Name                     Data
  cloud
           address=my-account-id.tmprl.cloud:7233
  default
  dev
           address=other
           tls=1
  local
           address=127.0.0.1:7233
           test=some


❯ ./temporal env list --output json
[
  {
    "Name": "default",
    "Data": {}
  },
  {
    "Name": "dev",
    "Data": {
      "address": "other",
      "tls": "1"
    }
  },
  {
    "Name": "local",
    "Data": {
      "address": "127.0.0.1:7233",
      "test": "some"
    }
  },
  {
    "Name": "cloud",
    "Data": {
      "address": "my-account-id.tmprl.cloud:7233"
    }
  }
]
```

thoughts?

Reply:
Overall this LGTM, I'd prefer `name`, and `variables` in lower case for the object keys.

Reply:
we have a flag `--fields long` that accomplishes similar results as `--verbose` in the PR

Reply:
adding `--fields` flag just for this case seems a bit extra. We've discussed with Roey and so far prefer this command to only output the names of the envirables, not values. For values better explicitly use `env get`

Reply:
Thank you @feedmeapples appreciate it.